torch/_torch_docs.py

```diff
@@ -3869,6 +3869,9 @@ def merge_dicts(*dicts):
 This is the second value returned by :meth:`torch.max`. See its
 documentation for the exact semantics of this method.
 
+.. note:: In case of multiple occurrences of the maximum values,
```

Comment:
"If there are multiple maximal values then the indices of the first maximal value are returned."

Reply:
Done.